ui/src/date_picker.tsx

```diff
@@ -68,7 +68,7 @@ export const
       onSelectDate = (d: Date | null | undefined) => {
         const val = (d === null || d === undefined) ? defaultVal : formatDate(d)
```

Comment:
nit: I know this is not part of the issue, but can you please refactor this? Thanks.
```suggestion
        const val = d ?? defaultVal : formatDate(d)
```